src/layers/GeoJson.js

```diff
@@ -49,7 +49,9 @@ class GeoJson extends Layer {
             opacityFactor,
         })
         this.addLayer(outlineLayer({ id, color: strokeColor, width }))
-        this.addLayer(lineLayer({ id, color, width }), { isInteractive })
+        this.addLayer(lineLayer({ id, color: strokeColor, width }), {
```

Comment:
I would suggest to use color: strokeColor || color

It will not break https://github.com/dhis2/maps-app/blob/dev/src/components/map/layers/TrackedEntityLayer.js#L69

Reply:
updated